ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/domain/Rejection.scala

```diff
@@ -12,17 +14,27 @@ import lf.data.Time.Timestamp
 import lf.transaction.GlobalKey
 import platform.server.api.validation.ErrorFactories
 import platform.store.appendonlydao.events.ContractId
-
 import com.google.protobuf.any.Any
 import com.google.rpc.error_details.ErrorInfo
 import com.google.rpc.status.Status
 
 private[sandbox] sealed trait Rejection extends Product with Serializable {
   def toStatus: Status
+  def completionInfo: CompletionInfo
```

Comment:
Add the completionInfo so we can generate `Update.CommandRejected` from `Rejection` without the need to thread `CompletionInfo` through by different means